src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="page">
+    <main class="page__content">
+      <div class="card" data-qa="card">
+        <img class="card__image" src="./images/imac.jpeg" alt="iMac">
+        <p class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+        <p class="card__code">Код товара: 195434</p>
+        <div class="card__rate">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="card__feedback">Отзывов: 5</div>
+        </div>
+        <div class="price card__price">
+          <p class="price__text">Цена:</p>
+          <p class="price__amount">69 999 грн</p>
+        </div>
+        <div class="card__buy">
+          <a href="#" class="btn" data-qa="hover">Купить</a>
+        </div>
+      </div>
+    </main>
+  </body>
+
+</html>
```

Comment:
```suggestion
</html>

```
потірбно додавати пустий рядок